specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ./images/
+
+= Import gluster cluster workflow
+
+== Problem description
+
+This specification describes import gluster cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+View design will look like:
+
+image::import.png[Import workflow]
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of clusters(imported and ready to import) and those clusters will be displayed on cluster list page.
+
+* A button will displayed for each cluster which are not imported and are ready to import.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* User will be redirected to new view.
+
+* UI will filter the available nodes for importing the cluster.
+
+* The view will list down the available nodes and a checkbox for enabling/disabling the cluster wide volume profiling.
+
+* Each node's name, IP address and role will be shown.
+
+* User can perform following operations on the node list.
+
+** filter by name, IP address and role.
+
+** sort by name, IP address and role.
+
+* User can check/uncheck if he wants to enable/disable the volume profiling.
+
+* All the available nodes will participate in the import cluster workflow.
+
+* User clicks the import button to begin the import operation.
+
+* Success message will be shown once import operation successfully started.
```

Comment:
@r0h4n If the cluster has some errors, the import button will be disabled for it on cluster list page. Hence the user can't see import cluster view for it. We can show cluster wide errors on cluster list page. 
Its being explained in [PR](https://github.com/Tendrl/specifications/pull/228/files) of cluster list as this scenario is handled in cluster list page.